lib/ui/home/home_page.dart

```diff
@@ -20,11 +21,11 @@ class _TabConfig {
 class HomePage extends HookWidget implements CobbleScreen {
   final _config = [
     _TabConfig(TestTab(), "Testing", RebbleIconsStroke.send_to_watch_checked),
-    _TabConfig(Placeholder(), "Health", RebbleIconsStroke.health),
+    _TabConfig(WidgetLibrary(), "Health", RebbleIconsStroke.health),
```

Comment:
Don't do this, show extra entry on setting screen when app is in debug mode, instead.

Reply:
Implemented, now settings will have extra entry if app is in debug mode, called "Widget Library".